### ISSUE TITLE
Permettre de choisir quels ajustements sont conservés ou non lors du renouvellement

### ISSUE THREAD

**Issue by thibaudgg** [acp-admin/acp-admin#72]
Dans certaines ACP, les ajustements (participations, prix panier à l'année, ...) doivent être conservés d'une année à l'autre alors que dans d'autres il ne faut pas les copier. Un réglage pour chaque ajustement serait utile.

**Comment by secretariatldc** [comment]
possibilité de le paramétrer par membres? :-p

**Comment by thibaudgg** [comment]
Non, cela me semble trop compliquer, je dirais uniquement globalement.

**Comment by thibaudgg** [comment]
Temps: ~1h
